bpf/bpf_sock.c

```diff
@@ -319,6 +314,49 @@ int sock4_xlate(struct bpf_sock_addr *ctx)
 	return SYS_PROCEED;
 }
 
+#if defined ENABLE_NODEPORT || defined ENABLE_EXTERNAL_IP
```

Comment:
nit: `defined(ENABLE_NODEPORT) || defined(ENABLE_EXTERNAL_IP)`

Reply:
I will change it! Wasn't sure which style to use, since we seem to have two different styles in our code, I found a lot of code which doesn't have the parenthesis either (e.g. `bpf_lxc.c`).

Reply:
Ah good point, might be good to clean this up at some point. If in doubt, always kernel style. :)